Instruments/HallProbe_ProLogix_new.py

```diff
@@ -1,56 +1,6 @@
 import socket
 import time
 
-class PrologixEthernet:
```

Comment:
I delete this because it's attempting to duplicate the functionality of `gpib_eth` which already exists and already works perfectly.